lib/yard-settings/handler.rb

```diff
@@ -0,0 +1,48 @@
+# :nodoc:
+class SettingsHandlerBase < YARD::Handlers::Ruby::Base
+  handles method_call :string_setting
+  handles method_call :number_setting
+  handles method_call :boolean_setting
+
+  namespace_only
+
+  def process
+    name = statement.parameters.first.jump(:tstring_content, :ident).source
+    object = YARD::CodeObjects::MethodObject.new(namespace, name)
+    register(object)
```

Comment:
@michaelklishin This is the place where I'd like your input on also creating "`read_timeout=`" as well (perhaps with the "Found using `method_missing`" addition, too?